### PR TITLE
Add a Maven badge to the README

### PR DESCRIPTION
**Pull request by cb372**
This makes it easy to check the latest published version

### CODE REVIEW

**Comment by DiegoVazquezNanini** [comment]
:+1: Thanks @cb372

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # configuration-magic
 _Where the magic happens_
 
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.gu/configuration-magic-core_2.11/badge.svg)]
```
Comment (adamnfish) [inline README.md:4]: Why the extra square brackets?

Comment (cb372) [inline README.md:4]: I was fixing it while you were writing that comment :)

Comment (adamnfish) [inline README.md:4]: Oops, sorry

**Pull request merged**